Microsoft.DotNet.Interactive.CSharp/LanguageServiceExtensions.cs

```diff
@@ -0,0 +1,50 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Linq;
+using System.Text;
+using Microsoft.CodeAnalysis.QuickInfo;
+using Microsoft.CodeAnalysis.Text;
+using Microsoft.DotNet.Interactive.LanguageService;
+
+namespace Microsoft.DotNet.Interactive.CSharp
+{
+    public static class LanguageServiceExtensions
+    {
+        public static Position ToLspPosition(this LinePosition linePos)
+        {
+            return new Position()
+            {
+                Line = linePos.Line,
+                Character = linePos.Character,
+            };
+        }
+
+        public static Range ToLspRange(this LinePositionSpan linePosSpan)
+        {
+            return new Range()
+            {
+                Start = linePosSpan.Start.ToLspPosition(),
+                End = linePosSpan.End.ToLspPosition(),
+            };
+        }
+
+        public static string ToMarkdownString(this QuickInfoItem info)
```

Comment:
How about introducing a return type that indicates that this is in fact a Markdown string?

Reply:
I'll switch it to return `MarkupContent` which specifies its type as `markdown` or `plaintext`.